src/main/java/ch/njol/skript/expressions/ExprGlowing.java

```diff
@@ -32,7 +32,7 @@
 @Name("Glowing")
 @Description("Indicates if targeted entity is glowing (new 1.9 effect) or not. Glowing entities can be seen through walls.")
 @Examples({"set glowing of player to true"})
-@Since("2.2-dev18")
+@Since("2.2-dev18, INSERT VERSION (toggle support)")
 public class ExprGlowing extends SimplePropertyExpression<Entity, Boolean> {
```

Comment:
This class's acceptChange doesn't accept TOGGLE changer, bug its change method does